Dockerfile

```diff
@@ -46,6 +47,8 @@ COPY --chown=nobody:nogroup ./ui/ui.go ./ui/ui.go
 COPY --chown=nobody:nogroup --from=ui-builder /app/packages/app/web/dist ./ui/packages/app/web/dist
 RUN go build -ldflags "-X main.version=${VERSION} -X main.commit=${COMMIT}" -trimpath -o parca ./cmd/parca
 RUN go install github.com/grpc-ecosystem/grpc-health-probe@latest
+# Predicatable path for copying over to final image
+RUN if [ "$(go env GOHOSTARCH)" != "$(go env GOARCH)" ]; then mv "$(go env GOPATH)/bin/$(go env GOOS)_$(go env GOARCH)/grpc-health-probe" "$(go env GOPATH)/bin/grpc-health-probe"; fi
 
 # this image is what docker.io/alpine:3.14.1 on August 13 2021
 FROM docker.io/alpine@sha256:be9bdc0ef8e96dbc428dc189b31e2e3b05523d96d12ed627c37aa2936653258c
```

Comment:
I kind of expected that we need to swap out the base image for the appropriate one with the target architecture? If not then I think I would need a comment for future Frederic to understand why :joy:

Reply:
It's a multi-arch container image already, see also: `buildah manifest inspect docker.io/alpine:3.15` Based on the hostarch it will pick the right top layer

Reply:
Woah, I didn't even know this worked with the exact SHA. Today I learned ...

Reply:
You might be right. Inspecting the sha gives a different output, I guess I'll should be able to find it out on an arm64 board over the weekend.